packages/lesswrong/components/posts/AddToCalendar/AddToCalendarIcon.tsx

```diff
@@ -20,10 +20,11 @@ const styles = (theme: ThemeType): JssStyles => ({
   }
 })
 
-const AddToCalendarIcon = ({post, label, hideTooltip, classes}: {
+const AddToCalendarIcon = ({post, label, hideTooltip, hidePlusIcon, classes}: {
```

Comment:
I used this in a previous iteration, seems fine to leave as an option even though it's not currently used.